packages/dds/merge-tree/src/client.ts

```diff
@@ -1014,7 +1053,7 @@ export class Client extends TypedEventEmitter<IClientEvents> {
 				break;
 			}
 			case MergeTreeDeltaType.OBLITERATE_SIDED: {
-				assert(false, "TODO: sided obliterate will come in a follow-up PR shortly.");
+				this.applyObliterateRangeOp({ op });
```

Comment:
i think this will lose the sequenced message is the remote op case. probably just pass the opargs like all the above functions

Reply:
`OBLITERATE_SIDED` now shares a case with `OBLITERATE`